### ISSUE TITLE
Missing options in Accessibility page

### ISSUE THREAD

**Issue by remyleone** [thenickdude/wacom-driver-fix#40]
I've installed the patched version of the software, but on the accessibility page of the Security and privacy menu I don't see any of : `PenTableDriver`, `WacomTabletDriver`, `TabletDriver` or `WacomTabletSpringboard`.
That being said, `PenTabletDriver` shows up in the monitoring page.

Setup: macOS 10.15.6
Tablet: CTH-661 bamboo fun

**Comment by thenickdude** [comment]
Tick the input monitoring one and reboot, that will likely trigger the
other one to appear when you try to use the tablet.

**Comment by remyleone** [comment]
Thanks a lot it worked :)
